draft-ietf-v6ops-464xlaton.xml

```diff
@@ -290,68 +270,82 @@ Therefore, as long as the host is not using DS-Lite, it MAY use 192.0.0.0/29 for
 The following recommendations are made instead:
 </t>
 <ul>
 <li>
 <t>
-In a single-address model, the CLAT instance SHOULD NOT obtain a dedicated /64 for the purpose of sending and receiving statelessly translated packets.
+If the node is extending IPv4 connectivity downstream and is not performing stateful NAT44 (so it needs to translate downstream IPv4 addresses to IPv6 statelessly), the node 
+SHOULD obtain a dedicated /64 prefix (e.g. via DHCPv6-PD).
 </t>
 </li>
 <li>
 <t>
-In a dedicated prefix model, the CLAT instance MAY do one of the following:
+If the CLAT instance only uses a single IPv4 address (including scenarios where the node is performing stateful NAT44 to that address), the instance SHOULD NOT obtain a dedicated /64 for the purpose of sending and receiving statelessly translated packets.
 </t>
-<ul>
-<li>
+</li>
+</ul>
+
 <t>
-Perform stateful NAT44 to translate all IPv4 addresses from the dedicated prefix to a single IPv4 address, then perform stateless CLAT.
+If the CLAT instance doesn't obtain a dedicated IPv6 prefix, the instance MUST obtain a dedicated IPv6 address used exclusively for CLAT functions. 
+This is needed to differentiate between inbound native IPv6 traffic and traffic which needs to be passed to the CLAT instance.
+For example:
 </t>
-</li>
+<ul>
 <li>
 <t>
-Obtain a dedicated IPv6 address for each CLAT IPv4 address.
+An ICMPv6 Echo Reply packet from 64:ff9b::203.0.113.1 can be a response to either an IPv6 ping to 64:ff9b::203.0.113.1, or an IPv4 ping to 203.0.113.1, translated by CLAT.
 </t>
 </li>
 <li>
 <t>
-Obtain a dedicated /64 prefix via DHCPv6-PD.
+An ICMPv6 error packet from a global IPv6 address (not belonging to the NAT64 prefix) might a response to either native IPv6 traffic from the host, or CLAT traffic (see <xref target="I-D.ietf-v6ops-icmpext-xlat-v6only-source"/> for more details).
 </t>
 </li>
 </ul>
-</li>
-</ul>
-
 <t>
-In a single-address model, the CLAT instance SHOULD obtain a dedicated IPv6 address used exclusively for CLAT functions. 
-This is needed to differentiate between inbound native IPv6 traffic and traffic which needs to be passed to the CLAT instance.
-For example:
+Using a dedicated IPv6 source address for CLAT traffic allows the node to make that distinction without keeping state, so CLAT can operate in the stateless mode (see Section 1.3 of <xref target="RFC7915"/>.
+In the absence of a dedicated address the CLAT instance would be required to use stateful NAT64.
+At the same time [RFC6877] defines 464XLAT as an architecure where the customer-side translation is stateless (it is even mentioned explictly in the [RFC6877] title.
+While it is possible to implement stateful client-side trabslation and use a single IPv6 address for both native and translated IPv6 packets, that approach doesn't fit the framework defined in [RFC6877] and, therefore, is out of scope of this document.
```

Comment:
Agreed! So... in that same spirit, should we specifically describe the use of NAT44 as a known possibility but not something in scope for this document? I think that could be accomplished with minor changes to existing phrasings (I've added a couple examples to this PR).

Reply:
Well...I'd be careful here: while NAT44 is not a great solution, do we really want to force every single device with tethering/internet sharing enabled to request /64? 

The text right here is useful because it justifies 'MUST ask for a dedicated IPv6 address' - I do not think the problem of '/64 vs one address + stateful NAT44' is as important as 'a dedicated address vs one address to rule them all'..